package/MDAnalysis/coordinates/FHIAIMS.py

```diff
@@ -0,0 +1,322 @@
+periodicity in that direction. In MDAnalysis, only the case of no
+lattice_vector or three lattice_vector lines are allowed.
+
+Atomic positions and names are specified either by the atom or by
+the atom_frac tags:
+
+    atom           x  y  z  name
+    atom_frac      nx ny nz name
+
+where x,y, and z are expressed in Angstrom, and nx, ny and nz are
+real numbers in [0,1] and are used to compute the atomic positions
+in units of the basic cell.
+
+The field name is a string identifying the atomic species.
+See also the specifications in the official `FHIAIMS format`_.
+
+Classes
+-------
+
+.. autoclass:: Timestep
+   :members:
+
+.. autoclass:: FHIAIMSReader
+   :members:
+
+.. autoclass:: FHIAIMSWriter
+   :members:
+
+
+Developer notes: ``FHIAIMSWriter`` format strings
+-------------------------------------------------
+
+The :class:`FHIAIMSWriter` class has a :attr:`FHIAIMSWriter.fmt`
+attribute, which is a dictionary of different strings for writing
+lines in ``.in`` files.  These are as follows:
+
+``xyz``
+  An atom line without velocities.  Requires that the 'name' and
+  'pos' keys be supplied.  E.g.::
+
+     fmt['xyz'].format(pos=(0.0, 1.0, 2.0), name='O')
+
+``box_triclinic``
+  The (optional) initial lines of the file which gives box dimensions.
+  Requires the box keyword, as a length 9 vector.  This is a flattened
+  version of the (3,3) triclinic vector representation of the unit
+  cell.
+
+.. _FHIAIMS: https://aimsclub.fhi-berlin.mpg.de/
+.. _FHIAIMS format: https://doi.org/10.6084/m9.figshare.12413477.v1
+
+"""
+from __future__ import absolute_import
+
+import re
+from six.moves import range, zip
+from six import raise_from
+
+import itertools
+import warnings
+
+import numpy as np
+
+from . import base
+from .core import triclinic_box, triclinic_vectors
+from ..exceptions import NoDataError
+from ..lib import util
+from ..lib import mdamath
+
+
+class Timestep(base.Timestep):
+
+    def _init_unitcell(self):
+        return np.zeros((3, 3), dtype=np.float32)
+
+    @property
+    def dimensions(self):
+        """unitcell dimensions (A, B, C, alpha, beta, gamma)
+        """
+        return triclinic_box(self._unitcell[0], self._unitcell[1], self._unitcell[2])
+
+    @dimensions.setter
+    def dimensions(self, new):
+        self._unitcell[:] = triclinic_vectors(new)
+
+
+class FHIAIMSReader(base.SingleFrameReaderBase):
+    """Reader for the FHIAIMS geometry format.
+
+    .. note::
+       Geometry files are single frame-only
+
+    """
+    format = ['IN', 'FHIAIMS']
+    units = {'time': None, 'length': 'Angstrom', 'velocity': None}
+    _Timestep = Timestep
+
+    def _read_first_frame(self):
+        with util.openany(self.filename, 'rt') as fhiaimsfile:
+            relative, positions, velocities, lattice_vectors = [], [], [], []
+            skip_tags = ["#","initial_moment"]
+            oldline=''
+            for line in fhiaimsfile:
+                line = line.strip()    # perhaps not needed
```

Comment:
Is needed because the [format doc](https://figshare.com/articles/FHI-aims_File_Format_Description_geometry_in/12413477/1) said "white space separated":
> In geometry.in, whitespace separates keywords and their values.  Empty lines are ignored andcomment lines can be inserted starting with the ’#’ character. 

Remove the comment